### PR TITLE
Task/add .nvmrc

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem
We have a Dockerfile that defines Node v12 as the version to use for this project, but there is no `.nvmrc` for those not using Docker when developing locally. We've had some issues recently when devs are on different versions of Node since v14 is now out & is the current latest version on installing Node.js.

# Solution
Add .nvmrc set to use Node v12, same as in Dockerfile. 

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1137-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1137-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1137-pr-onfido-sdk-ui-onfido.surge.sh

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+12
```
Comment (stefaniacardenas) [inline .nvmrc:1]: Shouldn't we use 14 instead (for docker too)? The issues with node 14 have been fixed in development

Comment (kopijunkie) [reply .nvmrc:1]: Have not been running on node 14 myself so far. But I can upgrade & double check everything still works fine then update .nvmrc & the Dockerfile.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1137-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1137-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1137-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**